### DISCUSSION TITLE
CEDS Anthropogenic ("SLCF") Emissions

### DISCUSSION THREAD

**Discussion by ssmithClimate** [PCMDI/input4MIPs_CVs#147] [General]
Some links related to differences between the CEDS data going onto ESGF shortly (e.g. CMIP6+) and the CMIP6 data.

Regional/sectoral differences:
https://github.com/JGCRI/CEDS/blob/master/documentation/Version_comparison_figures_v_2024_04_01_vs_2016_07_16(CMIP6).pdf

Major differences readme related to the above:
https://github.com/JGCRI/CEDS/wiki/Release-Notes

The major updates to spatial gridding:
  -- Large point sources, particularly SO2, more accurately located within countries
  -- New seasonality, including COVID 2020 changes and recovery in 2021+
  -- Updates to newer spatial proxies in general

Note that there will be further updates when we publish the final CMIP7 historical dataset at the end of this calendar year. For example, there have been signifiant updates to residential biomass consumption in the IEA energy statistics released 1-2 months ago.

**Comment by hegglinm** [comment]
Hi Steve, We are preparing for the PI control simulations. Any recommendation of how to use the SLCF emissions? Should we do the same as for the LLGHG and simply repeat 1850 values? Thanks, Michaela

**Reply by znichollscr** [reply]
Hi Michaela, as far as I can tell that is the advice (see also #169)

**Comment by pierrenabat** [comment]
Hello,
although supposed to be available for testing (see https://input4mips-controlled-vocabularies-cvs.readthedocs.io/en/latest/database-views/input4MIPs_delivery-summary_CMIP6Plus.html), several variables of CEDS-CMIP-2024-10-21 are no longer available for download on ESGF with this message:
"IMPORTANT! This dataset has been retracted and is no longer available for download."
Could you tell us what is the problem (can we test variables already downloaded before being retracted), and when a new version will  be available ?
Thanks in advance for your answer,
Pierre

**Reply by znichollscr** [reply]
Hi Pierre, good question and thanks for raising it. We are still finalising the details (see https://github.com/PCMDI/input4MIPs_CVs/pull/161) but, as I understand it, the issue is that negative numbers were found in some files when there shouldn't be any. So, I think you can use the files for testing data formats, naming etc. but I don't think I would use this for any model spin up or other such tests. The next version is in the publication queue I believe so, as I understand it, will hopefully be published and available this week.
Best,
Zeb

**Reply by pierrenabat** [reply]
Thanks for your answer !
Best,
Pierre

**Reply by znichollscr** [reply]
Hi Pierre, hopefully you've seen this already, but a new version of the dataset has been uploaded and is now available. For details, see https://input4mips-controlled-vocabularies-cvs.readthedocs.io/en/latest/dataset-overviews/anthropogenic-slcf-co2-emissions/ (#169 probably also worth following)

**Reply by pierrenabat** [reply]
Thank you for this information !

**Comment by vnaik60** [comment]
Hi Steve (@ssmithClimate),

A couple of questions on regional emissions comparing 'v20170519' (CMIP6) and 'v20241203' (CMIP6+): 

-  What is the source of the strong decline in residential (RCO) BC/OC emissions over China+ region in CMIP6+ versus CMIP6? This decline seems to be driving the decline in total BC/OC emissions over the China+ region. Perhaps this is a result of an update in the emission factors, activity data, aligment with local/regional inventories or a combination of all. Would be great if you could confirm.  
- What is the source of the decline in NOx emissions from the transportation sector over India? The [release notes](https://github.com/JGCRI/CEDS/wiki/Release-Notes) indicate that "India emissions now generally calibrated to GAINS emission factors", so this could be a potential source. 

Thank you so much! 
Best,
Vaishali

**Reply by ssmithClimate** [reply]
@vnaik60 
On BC/OC in China, there are two big updates. 

1. We updated our extrapolation of BC from coke production. That has gone done substantially in recent time due to technology changes. China (after some delay) successfully closed  all the traditional coke ovens that were highly polluting.
2. There were updates in data for biomass consumption with much lower biomass consumption in China over recent years. (And this data has been revised again, for a number of other countries, leading to lower residential sector emissions across the board in a number of countries other than China.)

As for India NOx, yes, that was the change to calibrating with GAINS EF. Significant uncertainty exists for India emissions overall, however.

**Reply by vnaik60** [reply]
Thanks Steve, this is helpful! I missed the decline in the energy sector. The decline shows up in the regional totals for China. 
[cmip6vscmip6plus.emissions.pdf](https://github.com/user-attachments/files/18740876/cmip6vscmip6plus.emissions.pdf)

**Comment by mattiarighi** [comment]
Dear @ssmithClimate,

great to see the updated CMIP6+ emissions!

We have noticed that for aircraft emissions the CMIP5 Lamarque et al. 2010 profiles, as drawn from Lee et al. are still used. Is there any plan to implement the distributions from the ELK-inventory we have been sharing with you?

The inventory is now documented in [this preprint on ESSD](https://essd.copernicus.org/preprints/essd-2025-454/essd-2025-454.pdf) and we are happy to provide feedback on the data.

Thank you for your efforts!

Best,
Mattia

**Comment by ssmithClimate** [comment]
Yes, I very much want to do that (and likely blend those two across a couple decades). But I don't know when we will have the resources to do that.
